smartsnippets/migrations/0006_auto_20180201_0301.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def update_smartsnippets_description(apps, schema_editor):
+    SmartSnippet = apps.get_model('smartsnippets', 'SmartSnippet')
+    SmartSnippet.objects.all().update(description=models.Func(models.F('description'),
+                                                   models.Value('Merlin'),
+                                                   models.Value('Media Manager'),
+                                                   function='replace'))
+    SmartSnippet.objects.all().update(description=models.Func(models.F('description'),
+                                                   models.Value('COVE'),
+                                                   models.Value('Media Manager'),
+                                                   function='replace'))
+
+
+def update_smartsnippets_name(apps, schema_editor):
+    SmartSnippet = apps.get_model('smartsnippets', 'SmartSnippet')
+    SmartSnippet.objects.all().update(description=models.Func(models.F('name'),
+                                                   models.Value('Merlin'),
+                                                   models.Value('Media Manager'),
+                                                   function='replace'))
+    SmartSnippet.objects.all().update(description=models.Func(models.F('name'),
+                                                   models.Value('COVE'),
+                                                   models.Value('Media Manager'),
+                                                   function='replace'))
+
+
+def revert(apps, schema_ditor):
```

Comment:
we should implement the revert, so our migrations be consistent and we'd be able to apply/unapply this.
typo: schema_ditor -> schema_editor

Reply:
I agree.